src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -1077,6 +1077,24 @@ class TestLibrary(SystemTests):
     def session_creation_kwargs(self):
         return {}
 
+    @pytest.mark.resource_name("4190/0")
+    @pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+    def test_lcr_compensation_data(self, session):
+        compensation_data = session.get_lcr_compensation_data()
+        session.configure_lcr_compensation(compensation_data)
```

Comment:
With NI-DCPower 23.3.0 this is returning `nidcpower.errors.DriverError: -1074135025: IVI: Invalid parameter.`

Reply:
This is the same issue as `test_lcr_custom_cable_compensation_data`: #1908. I have disabled this test and added a comment.